pkg/volume/local/local.go

```diff
@@ -218,6 +218,7 @@ func (plugin *localVolumePlugin) ConstructBlockVolumeSpec(podUID types.UID, volu
 		Spec: v1.PersistentVolumeSpec{
 			PersistentVolumeSource: v1.PersistentVolumeSource{
 				Local: &v1.LocalVolumeSource{
+					// Not needed because we don't need to detach local device from the host.
```

Comment:
- if the reconstructed volume is not needed to clean, the true source of the path from the kube-apiserver can be used
- if the reconstructed volume is needed to clean, this is not required because we don't need to detach local device from the host. all we need to clean is `GlobalMapPath` and `PodDeviceMapPath` which are determined by volume name and the pod UID.

Reply:
So we are not fixing fs on block reconstruction case?

Reply:
ah saw your initial comment. nm

Reply:
Yes, https://github.com/kubernetes/kubernetes/pull/84218 for block fs reconstruction